ci/live-pipeline.yml

```diff
@@ -170,15 +170,17 @@ jobs:
         status: success
         context: Deploying_Dev
         comment_file: GITHUB_OUTPUT/output.txt
+        delete_previous_comments: true
 
     on_failure:
       put: repo
       resource: pr-open
       params:
         path: repo
-        status: success
+        status: failure
```

Comment:
can these two changes also be made to the chunk on code starting on line 316 please